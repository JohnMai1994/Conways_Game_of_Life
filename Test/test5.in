new 10
init 5 0 5 1 5 2 5 3 5 4 5 5 5 6 5 7 5 8 5 9
-1 -1
print
step
print
step
print
step
print
step
print
steps 10
print
step
print
