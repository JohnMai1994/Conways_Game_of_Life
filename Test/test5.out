_____X____
_____X____
_____X____
_____X____
_____X____
_____X____
_____X____
_____X____
_____X____
_____X____
__________
____XXX___
____XXX___
____XXX___
____XXX___
____XXX___
____XXX___
____XXX___
____XXX___
__________
_____X____
____X_X___
___X___X__
___X___X__
___X___X__
___X___X__
___X___X__
___X___X__
____X_X___
_____X____
_____X____
____XXX___
___XX_XX__
__XXX_XXX_
__XXX_XXX_
__XXX_XXX_
__XXX_XXX_
___XX_XX__
____XXX___
_____X____
____XXX___
___X___X__
__X_____X_
__________
_X_______X
_X_______X
__________
__X_____X_
___X___X__
____XXX___
__________
__________
_____X____
____XXX___
__________
__________
____XXX___
_____X____
__________
__________
__________
__________
____XXX___
____XXX___
_____X____
_____X____
____XXX___
____XXX___
__________
__________
